docs/csharp/versioning.md

```diff
@@ -0,0 +1,128 @@
+---
+title: C# Versioning | C# Guide
+description: Understand how versioning works in C# and .NET
+keywords: .NET, .NET Core, C#
+author: tsolarin
+manager: wpickett
+ms.date: 2016/11/20
+ms.topic: article
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: aa8732d7-5cd0-46e1-994a-78017f20d861
+---
+
+# Versioning in C#
+
```

Comment:
Let's add a brief introduction here to tell readers what they will learn by reading this article. One or two paragraphs is all it should take.